### ISSUE TITLE
Deprecated type `np.int` used

### ISSUE THREAD

**Issue by tsbinns** [gordon-n-stevenson/colormap_to_ITKSNAP#1]
Hi @gordon-n-stevenson,

Thanks for creating this great tool, it really helped me out!

Not sure if this code is still being maintained, but there is an issue running with recent versions of NumPy.
Since v1.20 `np.int` is deprecated, and calling the `createLookupTable()` method raises the following error:
```
AttributeError: module 'numpy' has no attribute 'int'.
`np.int` was a deprecated alias for the builtin `int`. To avoid this error in existing code, use `int` by itself. Doing this will not modify any behavior and is safe. When replacing `np.int`, you may wish to use e.g. `np.int64` or `np.int32` to specify the precision. If you wish to review your current use, check the release note link for additional information.
The aliases was originally deprecated in NumPy 1.20; for more details and guidance see the original release note at:
    https://numpy.org/devdocs/release/1.20.0-notes.html#deprecations.
```

<br>

Fixing this is very simple, and just requires replacing `np.int` with `int` in this line:
https://github.com/gordon-n-stevenson/colormap_to_ITKSNAP/blob/d66208c1ce938deac3b4a51bdd88556a8dba8504/ColorMapWriter.py#L50

<br>

I would be very happy to open a PR with this change if you are interested.

Best,
Thomas

**Comment by gordon-n-stevenson** [comment]
Hi Thomas,

That would be great! Please do!

**Comment by gordon-n-stevenson** [comment]
Resolved by #2
